lua/entities/gmod_wire_expression2/core/custom/effects.lua

```diff
@@ -175,6 +175,7 @@ e2function effect effect:setAttachment(number index)
 end
 
 e2function void effect:play(string name)
+	if name == "dof_node" then return end
```

Comment:
This belongs in a separate pull request.

Reply:
As stated before it's actually already in a d that commit should be ignored if merged